libs/components/lookup/src/lib/modules/lookup/lookup.component.html

```diff
@@ -33,6 +33,7 @@
       (searchAsync)="onSearchAsync($event)"
       (selectionChange)="onAutocompleteSelectionChange($event)"
       (showMoreClick)="onShowMoreClick($event)"
+      (autocompleteOpenChange)="onAutocompleteOpenChange($event)"
```

Comment:
Do these changes need to be applied to the autocomplete cell editor, too?

Reply:
Good call. Added those.